quaternions/version.py

```diff
@@ -1,2 +1,2 @@
 # https://www.python.org/dev/peps/pep-0440/
-__version__ = '0.1.3.dev0'
+__version__ = '0.1.3.dev1'
```

Comment:
I'm sure @matiasg wants to release a stable version sooner than later :)

Reply:
yep, i think there are 2 more PRs pending, after which 0.1.4?